assets/liquid_staking/hipo.json

```diff
@@ -4,29 +4,39 @@
         "category": "liquid-staking",
         "subcategory": "",
         "website": "https://hipo.finance",
-        "description": "",
-        "organization": "hipo"
+        "description": "The #1 Most Profitable Liquid Staking on TON!",
+        "organization": "Hipo Finance"
     },
     "addresses": [
         {
             "address": "EQCLyZHP4Xe8fpchQz76O-_RmUhaVc_9BAoGyJrwJrcbz2eZ",
             "source": "",
-            "comment": "Hipo treasury",
+            "comment": "Hipo Treasury",
             "tags": [
                 "defi"
             ],
             "submittedBy": "ohld",
             "submissionTimestamp": "2025-01-15T07:07:33Z"
         },
         {
-            "address": "EQCSlnhRIHYCAlt1FoQVlBHzxmaQOMj9Efu80DwoggSlt_jN",
+            "address": "EQDPdq8xjAhytYqfGSX8KcFWIReCufsB9Wdg0pLlYSO_h76w",
```

Comment:
@shuva10v should we start to add labels for jetton masters of the projects as well?

Reply:
yes, we should. It will be useful for trading/tvl data